src/main/java/pacApp/pacConfig/WebSecurityConfig.java

```diff
@@ -91,19 +91,20 @@ protected void configure(HttpSecurity httpSecurity) throws Exception {
                 //CSRF is NOT needed because token is invulnerable
                 .csrf().disable()
                 .exceptionHandling().authenticationEntryPoint(unauthorizedHandler).and()
+                .logout().disable()
```

Comment:
The Spring Boot framework includes a default behavior for the logout process via the /logout endpoint. Since we implement the logout as part of a web service and in contrast to a web application, redirection to a specific URL is not desirable as an example.
[Spring Security returns 302](https://stackoverflow.com/questions/36366219/spring-security-returns-302-when-trying-to-logout)
[Spring disable logout redirect](https://stackoverflow.com/questions/36354405/spring-security-disable-logout-redirect)